versioned/tiered/dynamodb/src/test/java/com/dremio/nessie/versioned/impl/condition/TestExpressions.java

```diff
@@ -28,7 +28,6 @@
 
   private final Entity av0 = Entity.ofBoolean(true);
   private final Entity av1 = Entity.ofBoolean(false);
-  private final Entity av2 = Entity.ofString("mystr");
```

Comment:
Why changes to dynamo?

Reply:
We removed IF_NOT_EXISTS from ExpressionFunction.FunctionName as per Jacques comment on Dec 21
"Jacques Nadeau 1:58 PM
It looks like you're referring to the static SetClause.ifNotExists method. If we're not using it anywhere, yes, makes sense to remove. (edited) "

That broke the DynamoDB test which I fixed.
This was added as part of https://github.com/projectnessie/nessie/pull/741
This PR is based on #741 so includes the same change.